### PR TITLE
Stat series REST docs overhaul

### PR DESCRIPTION
**Pull request by KilimAnnejaro**
Rewriting the docs for the REST stat series endpoint.

### CODE REVIEW

File: api/rest/stat_series.md [hunk 1]
```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
-## GET Request
+## How to construct a request to the place statistics time series endpoint
 
-**Examples**
+### Step 1: Assembling the information you will need
 
-```bash
-curl 'https://api.datacommons.org/stat/series?place=geoId/06&stat_var=Count_Person_Male'
-```
+This endpoint requires two arguments and offers four additional optional arguments, as listed:
```
Comment (tjann) [inline api/rest/stat_series.md:42]: Maybe just "This endpoint requires two arguments:" and leave the additional optional as you have now "In addition to these required properties, this endpoint also.."

File: api/rest/stat_series.md [hunk 2]
```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/w32gmo68/8/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+ - `place`: For this parameter, you will need to specify the DCID (the unique ID assigned by Data Commons to each node in the graph) of the place you are interested in.
+ - `stat_var`: The statistical variable whose value you are interested in.
+
+In addition to these required properties, this endpoint also allows for other, optional arguments.
```
Comment (tjann) [inline api/rest/stat_series.md:47]: nit: arguments:

File: api/rest/stat_series.md [hunk 3]
```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
+
+  - `date`: If the property queried only takes on node values, you can use this argument to filter nodes in the response, ensuring the response only contains nodes with the specified type.
+
+  - `measurement_method`: You can specify this argument as `out` to indicate that you desire the response to only include nodes which are supercategories of the specified `DCIDs`, or `in` to only return nodes that are subcategories of the specified `DCIDs`. (For example, South America is a supercategory of Argentina, which in turn is a supercategory of Buenos Aires, as illustrated in Figure 1.)
+  
+  - `observation_period`: (≤ 500) Maximum number of values returned per node.
```
Comment (tjann) [inline api/rest/stat_series.md:53]: @KilimAnnejaro  I think these descriptions are leftover from prop-vals?

remove date and add scalingFactor?

File: api/rest/stat_series.md [hunk 4]
```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
-curl 'https://api.datacommons.org/stat/series?place=geoId/06&stat_var=Count_Person_Male'
-```
+This endpoint requires two arguments and offers four additional optional arguments, as listed:
 
-<iframe width="100%" height="300" src="//jsfiddle.net/datacommonsorg/w32gmo68/8/embedded/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
+ - `place`: For this parameter, you will need to specify the DCID (the unique ID assigned by Data Commons to each node in the graph) of the place you are interested in.
+ - `stat_var`: The statistical variable whose value you are interested in.
+
+In addition to these required properties, this endpoint also allows for other, optional arguments.
+
```
Comment (tjann) [inline api/rest/stat_series.md:48]: We may want to add a note somewhere about optional args.

Data Commons will return a default values for optional args, if you leave that argument blank. If you "overconstrain" by specifying a unit, scalingFactor, etc that does not exist for the place and variable, there will be no resulting time series.

Feel free to summarize/modify ^

File: api/rest/stat_series.md [hunk 5]
```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
-    "2013": 18726468,
-    "2014": 18911519,
-    "2015": 19087135,
-    "2016": 19200970,
-    "2017": 19366579,
-    "2018": 19453769,
-    "2011": 18387718,
-    "2012": 18561020
-  }
+  "value": integer
```
Comment (tjann) [inline api/rest/stat_series.md:67]: Is this right?

File: api/rest/stat_series.md [hunk 6]
```diff
@@ -34,69 +35,200 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
 }
 ```
 
-## Error Response
+## Example requests and responses
 
-### **Code**: `400 Bad Request`
+### Example 1: Retrieve the count of men in the state of California.
```
Comment (tjann) [inline api/rest/stat_series.md:73]: I think some of the jsfiddles are not for stat series. I'm also confused about the diff between the GET and CURL tabs. For stat series, we don't have POST, right?


@KilimAnnejaro I'll assume this section might have some pending changes. LMK when you're ready and I'll review again!

**Review by tjann — DISMISSED** [review]

File: api/rest/stat_series.md (hunk 1 above)
Comment (beets) [reply api/rest/stat_series.md:42]: In general, I think it's best not to enumerate the arguments for maintainability

File: api/rest/stat_series.md (hunk 4 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md:48]: I used the language we've had in other parts of the docs here. I also added language to indicate that too many optional args doesn't work out well.

**Comment by KilimAnnejaro** [comment]
Hi @tjann , there is a bug (also visible in the triple endpoint docs) where the tabs list is incorrect if you include tabs on some pages that are not included on other pages. I will work on fixing that this weekend.

File: api/rest/stat_series.md (hunk 6 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md:75]: Hi @tjann, thanks for these comments! As I mentioned in my message, there was a bug causing display issues here. I've fixed that (pending review) and you shouldn't see jsfiddles that are not relevant to this doc.

File: api/rest/stat_series.md (hunk 5 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md]: fixed!

File: api/rest/stat_series.md (hunk 3 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md]: sorry about that--fixed!

File: api/rest/stat_series.md [hunk 7]
```diff
@@ -39,21 +37,23 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
 
  - `place`: For this parameter, you will need to specify the DCID (the unique ID assigned by Data Commons to each node in the graph) of the place you are interested in.
  - `stat_var`: The statistical variable whose value you are interested in.
 
-In addition to these required properties, this endpoint also allows for other, optional arguments.
-
-  - `date`: If the property queried only takes on node values, you can use this argument to filter nodes in the response, ensuring the response only contains nodes with the specified type.
+In addition to these required properties, this endpoint also allows for other, optional arguments. Here are helpful arguments in regular use by Data Commons developers:
 
   - `measurement_method`: You can specify this argument as `out` to indicate that you desire the response to only include nodes which are supercategories of the specified `DCIDs`, or `in` to only return nodes that are subcategories of the specified `DCIDs`. (For example, South America is a supercategory of Argentina, which in turn is a supercategory of Buenos Aires, as illustrated in Figure 1.)
```
Comment (beets) [inline api/rest/stat_series.md:47]: There's an error here (in/out isn't for measurement_method). Can we update to refer to the glossary now that it's in?

File: api/rest/stat_series.md [hunk 8]
```diff
@@ -39,21 +37,23 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
   
   - `observation_period`: (≤ 500) Maximum number of values returned per node.
```
Comment (beets) [inline api/rest/stat_series.md:49]: This description is also incorrect, and we should point to the glossary

File: api/rest/stat_series.md [hunk 9]
```diff
@@ -39,21 +37,23 @@ You can find a list of StatisticalVariables with human-readable names [here](/st
...
 
   - `unit`: The unit of measurement.
 
+  - `scaling_factor`: Property of statistical variables indicating factor by which a measurement is multiplied to fit a certain format. For example, a proportion of 0.05 displayed as 5% has a scaling factor of 100, since 5 is equal to 0.05 multiplied by 0.05.
+
+Note that specifying arguments that do not exist for the target place and variable will result in an empty response.
```
Comment (beets) [inline api/rest/stat_series.md:55]: Can we call this out more? Perhaps the same treatment as the note at the end of the glossary.

File: api/rest/stat_series.md [hunk 10]
```diff
@@ -137,24 +148,29 @@ curl --request GET \
...
 
 {% endtab %}
 
 {% endtabs %}
 
-<script src="/assets/js/tabs.js"></script>
 </div>
 
-### Example 4: Retrieve the distrubtion of naloxone in Miami-Dade County in grams.
+### Example 4: Retrieve the distrubtion of the drug naloxone in Miami-Dade County in grams.
```
Comment (beets) [inline api/rest/stat_series.md:167]: Nit: typo for distribution

**Review by beets — CHANGES_REQUESTED** [review]
Thanks for the updates! Some requested changes below...

File: api/rest/stat_series.md (hunk 4 above)
Comment (beets) [reply api/rest/stat_series.md]: This is now split across three different places - can we centralize this, at the top perhaps?

File: api/rest/stat_series.md (hunk 7 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md:47]: Whoops, sorry about this! Fixing it.

File: api/rest/stat_series.md (hunk 8 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md:49]: fixed :sob:

File: api/rest/stat_series.md (hunk 9 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md]: added!

File: api/rest/stat_series.md (hunk 4 above)
Comment (KilimAnnejaro) [reply api/rest/stat_series.md]: I moved it to the top of the section. Let me know what you think!

**Review by beets — APPROVED** [review]

**Pull request merged**